src/api/ledger/parse/order-base.js

```diff
@@ -0,0 +1,27 @@
+/* @flow */
+'use strict';
+const utils = require('./utils');
+const parseAmount = require('./amount');
+const orderFlags = utils.core.Transaction.flags.OfferCreate;
+
+/*:: type Amount = string | {currency: string, issuer: string, value: string} */
+function parseOrder(takerGets: Amount, takerPays: Amount, flags: number):
+    Object {
+  const direction = (flags & orderFlags.Sell) === 0 ? 'buy' : 'sell';
+  const takerGetsAmount = parseAmount(takerGets);
+  const takerPaysAmount = parseAmount(takerPays);
+  const quantity = (direction === 'buy') ? takerPaysAmount : takerGetsAmount;
+  const totalPrice = (direction === 'buy') ? takerGetsAmount : takerPaysAmount;
+
+  return utils.removeUndefined({
+    direction: direction,
+    quantity: quantity,
+    totalPrice: totalPrice,
+    passive: ((flags & orderFlags.Passive) !== 0) || undefined,
```

Comment:
Yeah, omission would either mean "equal to the default value" or "equal to false", but these should be equivalent because false should always be default.